lighthouse-cli/index.js

```diff
@@ -139,22 +139,34 @@ if (cli.verbose) {
   flags.logLevel = 'error';
 }
 
-// kick off a lighthouse run
-lighthouse(url, flags, config)
-  .then(results => Printer.write(results, outputMode, outputPath))
-  .then(results => {
-    if (outputMode !== 'html') {
-      Printer.write(results, 'html', './last-run-results.html');
-    }
+function runLighthouse(addresses) {
+  // Process URLs once at a time
+  let address = addresses.shift();
+  if (!address) {
     return;
-  })
-  .catch(err => {
-    if (err.code === 'ECONNREFUSED') {
-      console.error('Unable to connect to Chrome. Please run Chrome w/ debugging port 9222 open:');
-      console.error('    npm explore -g lighthouse -- npm run chrome');
-    } else {
-      console.error('Runtime error encountered:', err);
-      console.error(err.stack);
-    }
-    process.exit(1);
-  });
+  }
+
+  lighthouse(address, flags, config)
+    .then(results => Printer.write(results, outputMode, outputPath))
+    .then(results => {
+      if (outputMode !== 'html') {
+        Printer.write(results, 'html', './last-run-results-' + Math.random() + '.html');
```

Comment:
What do folks think of just slugifying whatever comes in so:
`last-run-results-example-com-page-1.html`?

If we're worried about multiple runs, could do slugify + timestamp + .html.